src/asciidoc/docs/refs/glossary.adoc

```diff
@@ -0,0 +1,59 @@
+= Glossary
+:page-slug: glossary
+:page-description: Definitions of Koji-related terms.
+
+context::
+A state or mode of a Koji app.
+The default mode is the first screen displayed when the user starts the app.
+Learn more: <<intro-to-contexts#>>
+
+creator::
+Someone who creates media and other assets on social media and posts Koji apps to their Koji profile.
+Also see *developer* and *user*.
+
+developer::
+Someone who writes code to create Koji apps.
+Also see *creator* and *user*.
+
+entitlement::
+Permissions that allow the Koji app to use certain platform features, such as enabling customization or defining product information for in-app purchases.
+
+in-app purchase (IAP)::
+A financial transaction made within a Koji app.
+
+Koji app::
+A mini web application that can be embedded in a Koji profile, shared on social media platforms, embedded in any website, or sent via chat messengers or email.
+Learn more: <<introduction#>>
+
+Koji App Store::
+The Koji marketplace for apps. After you publish your app, you can create a listing for it in the Koji App Store to increase its exposure to Koji users.
+Learn more: <<app-store#>>
+
+Koji profile::
+A social media creator's link-in-bio which consolidates the creator's online presence.
```

Comment:
A quick Google shows that "link in bio" (lowercase) is the most common form. I'm putting the phrase between quotation marks, otherwise, it would sound like "creator's link which is in their bio". 

I think the form will eventually standardize industry-wide, as the phrase gets more traction.